ptc/account_generation/config.py

```diff
@@ -66,6 +66,7 @@ class Config(BaseModel):
     additional_sleep_time: int = 0
     webdriver_max_wait: int = 30
     network_blocks: Optional[List[str]] = None
+    max_accounts_per_domain: int = 100  # Default max accounts per domain
```

Comment:
not a big fan of this default, default should be no limit (-1 or something)

Reply:
I didn't realize it had a fallback default.  I can set it to 999999 like I suggested in the PR.  Or do you think people will make more than 1 million per domain?

Reply:
make the default to -1 
and before this part of the code

```python
max_accounts = config.max_accounts_per_domain
if not check_domain_account_limit(domain, max_accounts, accounts_collection):
        logger.error(f"Domain '{domain}' has reached the maximum allowed accounts ({max_accounts}). No account will be created.")
        return False
```

add this:

```python
max_accounts = config.max_accounts_per_domain
if max_accounts:
    if not check_domain_account_limit(domain, max_accounts, accounts_collection):
        logger.error(f"Domain '{domain}' has reached the maximum allowed accounts ({max_accounts}). No account will be created.")
        return False

Reply:
instead of erroring when the account reaches limit, could it just try a different domain?

likewise, similar to the `show_total_accounts` config option, would it make sense to print the summary by account on start up if configured?

Reply:
This should be handled with my latest push.